scripts/changelog.js

```diff
@@ -0,0 +1,13 @@
+import { getReleaseLine } from "@svitejs/changesets-changelog-github-compact";
```

Comment:
I don't see `getReleaseLine` exported in the ESM (or CJS) version of this plugin. And I'm not sure that ESM is supported for our changesets workflow.

I might suggest CJS instead with something along the lines of:

```js
const { default: changelogFunctions } = require('@svitejs/changesets-changelog-github-compact');

const customFunctions = {
  ...changelogFunctions,
  getDependencyReleaseLine: async () => ""
};

module.exports = {
  ...customFunctions,
  default: customFunctions
};
```

?

Reply:
That's a good point on the CJS bit- I thought I'd seen ESM used in changelog.js files in our other repos, but I was 100% wrong about that. Will implement your recommended changes.